src/main/scala/nl.knaw.dans.easy.bagstore/component/BagStoresComponent.scala

```diff
@@ -50,6 +50,19 @@ trait BagStoresComponent {
         }
     }
 
+    def getStream(itemId: ItemId, output: => OutputStream, fromStore: Option[Path] = None): Try[Unit] = {
```

Comment:
The line above is the only difference with the `get` method. I don't understand the magic. Do you really need to return a Unit? At first sight I'd expect a stream.